fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/ParameterDAO.java

```diff
@@ -1,24 +1,27 @@
 /*
- * (C) Copyright IBM Corp. 2017,2019
+ * (C) Copyright IBM Corp. 2017,2018,2019
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 package com.ibm.fhir.persistence.jdbc.dao.api;
 
+import java.sql.Array;
+import java.sql.Connection;
 import java.util.List;
+import java.util.Map;
 
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
 import com.ibm.fhir.persistence.jdbc.dto.Parameter;
 import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDBConnectException;
 import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDataAccessException;
 
 /**
- * This Data Access Object interface defines methods for creating, updating, and retrieving rows in the FHIR Parameter table.
- * @author markd
- *
+ * This Data Access Object interface defines methods for creating, updating, 
+ * and retrieving rows in the IBM FHIR Server parameter-related tables.
  */
 public interface ParameterDAO extends FHIRDbDAO {
-
+    
```

Comment:
spaces

Reply:
Eclipse does this.  We're wildly inconsistent as to whether empty lines have spaces that indent it to the length of the previous/next line or not.

Reply:
let me know if you find a way to have eclipse not add the spaces on newlines...